homeassistant/components/assist_satellite/__init__.py

```diff
@@ -2,33 +2,93 @@
 
 import logging
 
+import voluptuous as vol
+
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.core import HomeAssistant
+from homeassistant.core import HomeAssistant, SupportsResponse
 from homeassistant.helpers import config_validation as cv
 from homeassistant.helpers.entity_component import EntityComponent
 from homeassistant.helpers.typing import ConfigType
 
 from .const import DOMAIN
 from .entity import AssistSatelliteEntity
-from .models import AssistSatelliteState
+from .models import (
+    AssistSatelliteEntityFeature,
+    AssistSatelliteState,
+    PipelineRunConfig,
+    PipelineRunResult,
+)
 
 __all__ = [
     "DOMAIN",
-    "AssistSatelliteState",
     "AssistSatelliteEntity",
+    "AssistSatelliteEntityFeature",
+    "AssistSatelliteState",
+    "PipelineRunConfig",
+    "PipelineRunResult",
+    "SERVICE_WAIT_WAKE",
+    "SERVICE_GET_COMMAND",
+    "SERVICE_SAY_TEXT",
+    "ATTR_WAKE_WORDS",
+    "ATTR_PROCESS",
+    "ATTR_ANNOUNCE_TEXT",
 ]
 
 _LOGGER = logging.getLogger(__name__)
 
 PLATFORM_SCHEMA_BASE = cv.PLATFORM_SCHEMA_BASE
 
+ATTR_WAKE_WORDS = "wake_words"
+ATTR_PROCESS = "process"
+ATTR_ANNOUNCE_TEXT = "announce_text"
+
+SERVICE_WAIT_WAKE = "wait_wake"
+SERVICE_GET_COMMAND = "get_command"
+SERVICE_SAY_TEXT = "say_text"
+
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     component = hass.data[DOMAIN] = EntityComponent[AssistSatelliteEntity](
         _LOGGER, DOMAIN, hass
     )
     await component.async_setup(config)
 
+    component.async_register_entity_service(
```

Comment:
This should not be a service. It should be a websocket API to be used by the wizard.

Reply:
It should also be its very own PR. Each service should be its own service.